addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -229,6 +227,12 @@ var SnippetEditor = Widget.extend({
         }
         return _t("Block");
     },
+    /**
+     * @returns {boolean}
+     */
+    isTargetInAnimationPreview: function () {
+        return this.$target && this.$target.hasClass('o_animate_preview');
+    },
```

Comment:
Two problems with this:
- We lose the fact the overlay reposition correctly on any animation/transition (header effects / any other effect / code from the community / etc)
- You should not reference a class of another app in an app: `o_animate_preview` comes from website_animate, which is not a dependency of website so you cannot reference it in website.

I don't know about a good solution to fix all of this generically, at worst we can make a specific fix like yours but then it should be in website_animate :wink:

Reply:
You are totally right :) 
Further more I have found out this was not the right selector... The class used now is 'o_we_overlay_preview' which is part of web_editor.